runtime/v1/linux/runtime.go

```diff
@@ -391,7 +393,17 @@ func (r *Runtime) loadTasks(ctx context.Context, ns string) ([]*Task, error) {
 		} else {
 			go io.Copy(ioutil.Discard, shimStderrLog)
 		}
-
+		go func() {
+			select {
+			case <-shimExit:
```

Comment:
```
runtime/v1/linux/runtime.go:397:4:warning: should use a simple channel send/receive instead of select with a single case (S1000) (staticcheck)
```

please update it~